.github/actions/windows-signing/config_x64.toml

```diff
@@ -34,15 +35,6 @@ exclude = [
     "obs.pdb",
 ]
 
-## Delta patch generation
-[generate]
-patch_type = "zstd"
-compress_files = true
-
-exclude_from_parallel = [
-    "libcef.dll"
-]
-
```

Comment:
What's the reason for this removal?

Reply:
This only affects delta patches, which this job no longer creates.